src/Lecture2.hs

```diff
@@ -185,7 +284,15 @@ False
 True
 -}
 isIncreasing :: [Int] -> Bool
-isIncreasing = error "TODO"
+isIncreasing [] = True
+isIncreasing (x : xs) = go x xs
+ where
+  go _ [] = True
+  go prev (a : as)
+    | prev > a = False
+    | otherwise = go a as
```

Comment:
You can implement `isIncreasing` in a slightly more elegant way by relying on laziness in Haskell 🦥 

```suggestion
isIncreasing [_] = True
isIncreasing (x : y : ys) = x < y && isIncreasing (y : ys)
```